src/components/screens/votingQueue/editor/editor.js

```diff
@@ -44,17 +68,20 @@ const normalizeVotesForTx = votes =>
  * @param {Function} t - i18n translation function
  * @returns {Object} The feedback object including error status and messages
  */
+// eslint-disable-next-line max-statements
 const validateVotes = (votes, balance, fee, t) => {
   const messages = [];
-
   const areVotesInValid = Object.values(votes).some(vote =>
     (vote.unconfirmed === '' || vote.unconfirmed === undefined));
+  const votesStats = getVoteStats(votes);
 
   if (areVotesInValid) {
     messages.push(t('Please enter vote amounts for the delegates you wish to vote for'));
   }
 
-  if (Object.keys(votes).length > 10) {
+  if (Object.keys(votesStats.added).length > 10
+    || Object.keys(votesStats.edited).length > 10
+    || Object.keys(votesStats.removed).length > 10) {
```

Comment:
A user can't have more than 10 votes at a time. if you are editing or removing votes, it's an action on the already voted delegates which can't be more than 10. you need to make sure after submission, there's only 10 votes left. Something like:
```
Object.keys(votesStats.added).length + Object.keys(votesStats.edited).length <= 10
```